### PR TITLE
NEXT-37776 - Move SDK plugin tests

### PR DESCRIPTION
**Pull request by seggewiss**
## What?

Add the SDK TestPlugin from platform to meteor and run e2e tests here.

## Testing?

Playwright

### CODE REVIEW

File: examples/TestPlugin/tests/acceptance/tests/context/context.spec.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        //  await ShopAdmin.goesTo(AdminDashboard.url());
+        // or that?
+        await AdminPage.goto(AdminDashboard.url());
+
+        // No loader or skeleton should be visible
+        await expect(page.locator('.sw-loader')).toBeHidden();
+        await expect(page.locator('.sw-skeleton')).toBeHidden();
+
+        // The main hidden iFrame should exist
+        const mainHidden = await getSDKiFrame(page, 'sw-main-hidden');
```
Comment (pweyck) [inline examples/TestPlugin/tests/acceptance/tests/context/context.spec.ts:39]:
```suggestion
        const mainHidden = await getSDKiFrame(ShopAdmin.page, 'sw-main-hidden');
```

File: examples/TestPlugin/tests/acceptance/tests/context/context.spec.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+        // Go to dashboard
+        // This
+        //  await ShopAdmin.goesTo(AdminDashboard.url());
+        // or that?
+        await AdminPage.goto(AdminDashboard.url());
+
+        // No loader or skeleton should be visible
+        await expect(page.locator('.sw-loader')).toBeHidden();
+        await expect(page.locator('.sw-skeleton')).toBeHidden();
```
Comment (pweyck) [inline examples/TestPlugin/tests/acceptance/tests/context/context.spec.ts:36]:
```suggestion
```
Not required, ShopAdmin fixture should handle that

File: examples/TestPlugin/tests/acceptance/tests/context/context.spec.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    }) => {
+
+    });
+
+    test('@sdk: get current language', async ({
+        AdminPage,
+        ShopAdmin
+        page,
+        TestDataService,
+        AdminDashboard
```
Comment (pweyck) [inline examples/TestPlugin/tests/acceptance/tests/context/context.spec.ts:22]:
```suggestion
        ShopAdmin,
        TestDataService,
        AdminDashboard,
```

File: examples/TestPlugin/tests/acceptance/tests/context/context.spec.ts [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    }) => {
+        // Create product and assign id
+        const product = await TestDataService.createBasicProduct();
+        productId = product.id;
+
+        // Go to dashboard
+        // This
+        //  await ShopAdmin.goesTo(AdminDashboard.url());
+        // or that?
+        await AdminPage.goto(AdminDashboard.url());
```
Comment (pweyck) [inline examples/TestPlugin/tests/acceptance/tests/context/context.spec.ts:32]:
```suggestion
        await ShopAdmin.goto(AdminDashboard.url());
```

File: examples/TestPlugin/tests/acceptance/tests/window/window.spec.ts [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+        await ShopAdmin.page.locator('.sw-admin-menu__item--sw-order').click();
+        await ShopAdmin.page.getByRole('link', { name: 'Test item' }).first().click();
+    
+        const frame = await getSDKiFrame(ShopAdmin.page, 'ui-main-module-add-main-module');
+        await frame.locator('button', { hasText: 'Dispatch a notification' }).click();
+    
+        // Click the button that opens the new tab
+        await ShopAdmin.page.locator('.sw-button__content', { hasText: 'Redirect to Shopware' }).click();
+        // Wait for the new page to be created
+        const pagePromise = context.waitForEvent('page');
```
Comment (pweyck) [inline examples/TestPlugin/tests/acceptance/tests/window/window.spec.ts:26]: This should be registered before the click

File: examples/TestPlugin/tests/acceptance/tests/window/window.spec.ts [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+    
+        // Wait for the new page to be created and load
+        const newPage = await pagePromise;
+    
+        // Check the URL of the new page
+        expect(newPage.url()).toBe('https://www.shopware.com/');
```
Comment (pweyck) [inline examples/TestPlugin/tests/acceptance/tests/window/window.spec.ts:32]:
```suggestion
        // get promise for new page event
        const pagePromise = ShopAdmin.page.context().waitForEvent('page');
        
        // Click the button that opens the new tab
        await ShopAdmin.page.locator('.sw-button__content', { hasText: 'Redirect to Shopware' }).click();
    
        // Wait for the new page to be created and load
        const newPage = await pagePromise;
    
        // Check the URL of the new page
        expect(newPage.url()).toBe('https://www.shopware.com/');
```

**Review by pweyck — APPROVED** [review]

**Review by tajespasarela — APPROVED** [review]

**Pull request merged**